src/ripple/app/ledger/InboundLedger.cpp

```diff
@@ -67,6 +67,16 @@ InboundLedger::InboundLedger (uint256 const& hash, std::uint32_t seq, fcReason r
         "Acquiring ledger " << mHash;
 }
 
+void InboundLedger::update (std::uint32_t seq)
+{
+    ScopedLockType sl (mLock);
+
+    if ((seq != 0) && (mSeq == 0))
+        mSeq = seq;
+
+    touch ();
```

Comment:
I'll go against the prevailing wind of fewer characters is better.  I was reading through the code and wondered what `touch()` did.  It's not a method in this class, so it's either a free function or in one of the (three) base classes.  It turns out that `touch()` is in the `PeerSet` base class and changes its `mLastAction` member.  Given the time it took to find that answer, I'd be inclined to write this call as `PeerSet::touch()` as a courtesy to the next reader.  Just my opinion.